usr/local/share/bastille/list.sh

```diff
@@ -189,6 +174,23 @@ get_jail_info() {
     if [ "$(grep -c "^linprocfs" "${bastille_jailsdir}/${JAIL_NAME}/fstab" 2> /dev/null)" -gt 0 ]; then IS_LINUX_JAIL=1; fi
```

Comment:
I still consider this wrong because it could be manually mounted by the admin for the Linux binary compat: `/compat/linux` and not `/`
Only this constitutes a Linux jail:
```
        echo -e "linprocfs       ${bastille_jail_path}/proc     linprocfs       rw                      0       0" >> "${bastille_jail_fstab}"
```
nothing else

Reply:
We should leave this uncertainty for now.